### PR TITLE
Update RTA common.py for py3

### PR DESCRIPTION
**Pull request by brokensound77**
## Summary
- remove py2 compat
- add type hinting
- format

### CODE REVIEW

**Review by Mikaayenson — APPROVED** [review]
LGTM.

File: rta/common.py [hunk 1]
```diff
@@ -198,14 +195,23 @@ def close():
...
+    hide_log=False,
+    mute=False,
+    timeout: int = 30,
+    wait=True,
+    kill=False,
+    drop=False,
+    stdin: Optional[Union[bytes, str]] = None,
+    shell=False,
+    **kwargs,
+):
```
Comment (Mikaayenson) [inline rta/common.py:209]: I wish black didn't do this but it's all good.

**Review by Mikaayenson — APPROVED** [review]
LGTM

File: rta/common.py (hunk 1 above)
Comment (brokensound77) [reply rta/common.py:209]: agree - I am torn on some of the formatting, but should be fine for this as a cleanup 👍

**Review by terrancedejesus — APPROVED** [review]
Looks great!

**Pull request merged**